test/test.sh

```diff
@@ -1,30 +1,58 @@
-#!/bin/bash -e
+#!/bin/sh
+
+set -eu
 
 export TZ='US/Pacific'
 
-for pcap in `ls test/pcap`
-do
+cleanup()
```

Comment:
what if someone wanted to inspect the output after the test was run?  do you have a scenario you are concerned about that requires us to do the temp cleanup?

Reply:
well I had 40 /tmp/tmp.* files on my server from tests :)

The quickest way is to modify the script to set TMP to something else. We can also add a flag which prevents cleanup during development.

Reply:
fair enough.  how about only calling cleanup as the very last step and not trapping.  that way temp files are only cleaned up in the event of test success.  if there is a failure the files are left for someone to inspect

Reply:
Yeah that's maybe not a bad idea as well, as it should never fail to begin with. I agree.